Database upgrade failed

Upgraded from S4670 to S4700 today.  Noticed the following error and failure during the database upgrade.

```
Preparing to unpack .../openhab_5.0.0~S4700-1_all.deb ...
Unpacking openhab (5.0.0~S4700-1) over (5.0.0~S4670-1) ...
Preparing to unpack .../openhab-addons_5.0.0~S4700-1_all.deb ...
Unpacking openhab-addons (5.0.0~S4700-1) over (5.0.0~S4670-1) ...
Setting up openhab (5.0.0~S4700-1) ...

[openHAB] Listing important changes for version 5.0.0:
(removed for brevity)

[openHAB] Running JSON Database upgrade tool (${OPENHAB_RUNTIME}/bin/upgradetool.jar).
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/gson/JsonSyntaxException
        at org.openhab.core.tools.UpgradeTool.main(UpgradeTool.java:117)
Caused by: java.lang.ClassNotFoundException: com.google.gson.JsonSyntaxException
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        ... 1 more
[openHAB] WARNING: internal database upgrade tool failed.
[openHAB] please check the openHAB website (www.openhab.org) for upgrade requirements.
[openHAB] openHAB was not running so will not start after upgrade.
[openHAB] Please use the command:
            sudo /bin/systemctl start openhab.service


```

--- a/tools/upgradetool/pom.xml
+++ b/tools/upgradetool/pom.xml
@@ -92,6 +92,12 @@
       <version>${slf4j.version}</version>
       <scope>compile</scope>
     </dependency>
+    <!-- do not remove, inherited dependency gson from core.storage.json is not provided via runtime bom -->
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.11.0</version>
+    </dependency>
   </dependencies>
 
   <build>

